src/test/kotlin/javabot/operations/KarmaOperationTest.kt

```diff
@@ -6,18 +6,68 @@ import java.util.Date
 import javabot.BaseTest
 import javabot.dao.KarmaDao
 import javabot.dao.NickServDao
+import javabot.dao.util.EntityNotFoundException
 import javabot.registerIrcUser
 import javax.inject.Inject
 import org.testng.Assert.assertEquals
 import org.testng.Assert.assertFalse
 import org.testng.Assert.assertTrue
+import org.testng.annotations.DataProvider
 import org.testng.annotations.Test
 
 @Test(groups = arrayOf("operations"))
-class KarmaOperationTest
+class sKarmaOperationTest
```

Comment:
whut?

Reply:
Errant keystroke, fixed in a subsequent commit.